pkg/controller/storagecluster/cephcluster.go

```diff
@@ -343,8 +340,12 @@ func newExternalCephCluster(sc *ocsv1.StorageCluster, cephImage string, monitori
 	return externalCephCluster
 }
 
+func getMinNodes(sc *ocsv1.StorageCluster) int {
+
+	return defaults.DeviceSetReplica
+}
+
 func getMonCount(nodeCount int) int {
-	count := defaults.MonCountMin
```

Comment:
This removes the only reference to this variable in the code. Either keep this logic, or remove the MonCountMin variable.

Reply:
Done